package.json

```diff
@@ -19,7 +19,8 @@
     "contributors:add": "npx all-contributors add",
     "contributors:generate": "npx all-contributors generate",
     "test:ci": "npx cross-env NODE_ENV=build npm run test && npm run test:jest",
-    "build:lib": "npx ng build spectator"
+    "build:lib": "npx ng build spectator && npm run build:schematics",
+    "build:schematics": "tsc -p projects/spectator/schematics/tsconfig.json && cp projects/spectator/schematics/src/collection.json dist/spectator/schematics/ && cp projects/spectator/schematics/src/ng-add/schema.json dist/spectator/schematics/ng-add/"
```

Comment:
I do not know how to implement building and copying schematics files natively for angular.json. So I just created it as is for to find solution together.

Reply:
Check out https://github.com/angular/angular-cli/blob/master/docs/documentation/stories/asset-configuration.md#writing-assets-outside-of-dist.

Reply:
I thought about it too, but the ng-packagr builder does not have the `assets` config in angular.json

Reply:
So check out Akita's version. https://github.com/datorama/akita/blob/master/package.json#L17

Reply:
Just copy all of source files (ts+js)? I will make commit with it soon.